src/mapping/query/functions.rs

```diff
@@ -376,34 +409,17 @@ impl Function for Md5Fn {
 //------------------------------------------------------------------------------
 
 #[derive(Debug)]
-pub(in crate::mapping) struct NowFn {
-    query: Box<dyn Function>,
-}
```

Comment:
Ignore this part, shouldn't have been part of this PR (see https://github.com/timberio/vector/issues/3753#issuecomment-692749276), will remove.